frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -232,7 +234,11 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
     const selectedData: NodeKind[] = _.filter(nodes, 'selected');
     const promises = [];
 
-    promises.push(...makeLabelNodesRequest(selectedData));
+    if(selectedData.length !== 0) {
```

Comment:
This logic will no longer be needed after modifying the logic for label checking as per above comments.